src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-hptext.html

```diff
@@ -0,0 +1,164 @@
+
+      #remove-banners {
+        margin-top: 16px;
+        background-color: var(--paper-red-a200);
+        color: white;
+      }
+
+      paper-fab {
+        position: absolute;
+        right: 24px;
+        bottom: 24px;
+        --paper-fab-background: var(--lancie-button-color);
+      }
+
+      /*TODO: Should be in global scope*/
+
+      code {
+        color: #D72B3F;
+        background: #F7F7F9;
+        border-radius: 5px;
+        padding: 5px;
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="bannersAjax" refurl="web/banners" last-response="{{banners}}"></lancie-ajax>
+    <lancie-ajax id="deleteBannersAjax" refurl="web/banners" method="DELETE"></lancie-ajax>
+
+    <lancie-admin-page-layout endpoint="banner" on-banner-removed="refreshBanners">
+      <p>Here is where you will be able to schedule the text of the banner.</p>
+      <p>Requests will be on
+        <code>/api/v1/web/banners</code>
+      </p>
+
+      <div class="timeline-container">
+        <ul>
+          <template is="dom-repeat" items="{{banners}}">
+            <li>
+              <span></span>
+              <div>
+                <div class="text">{{item.text}}</div>
+              </div>
+              <span class="number">
+                <span>{{item.startDate}}</span>
+                <span>{{item.endDate}}</span>
+              </span>
+            </li>
+          </template>
+        </ul>
+      </div>
+
+      <paper-button id="remove-banners" raised on-tap="_openRemoveBannersDialog">REMOVE ALL</paper-button>
+
+      <paper-dialog id="removeBannersDialog" entry-animation="scale-up-animation"
+                    exit-animation="fade-out-animation">
+        <h2>Remove all scheduled banners</h2>
+        <p>Are you sure that you want to remove all scheduled banners?</p>
+        <div class="buttons">
+          <paper-button dialog-dismiss>No</paper-button>
+          <paper-button dialog-confirm autofocus on-tap="_removeBanners">Yes, I am sure!</paper-button>
+        </div>
+      </paper-dialog>
+
+      <div class="scheduler-container">
+        <template is="dom-repeat" items="{{banners}}">
+          <lancie-admin-banner-scheduler banner="{{item}}"></lancie-admin-banner-scheduler>
+        </template>
+      </div>
+
+      <paper-fab icon="lancie-admin:add" on-tap="addBanner"></paper-fab>
+    </lancie-admin-page-layout>
+  </template>
+
+  <script>
+    class LancieAdminHpText extends Polymer.Element {
+      static get is() {
+        return 'lancie-admin-hptext';
+      }
+
+      static get properties() {
+        return {
+          banners: Array,
+        };
+      }
+
+      addBanner() {
+        let latest = this.getLatestDate();
+        this.push('banners', {startDate: latest, endDate: latest});
+      }
+
+      refreshBanners() {
+        this.$.bannersAjax.generateRequest();
+        this.sortOnStartDate();
+      }
+
+      sortOnStartDate() {
+        this.banners.sort((x, y) => x.startDate - y.startDate);
```

Comment:
Does this trigger the bindings update? Else it has to be `this.banners = this.banners.sort`